src/main/src/main/java/org/geoserver/catalog/HTTPStoreInfo.java

```diff
@@ -65,4 +65,8 @@
      * @see #setMaxConnections(int)
      */
     public void setUseConnectionPooling(boolean useHttpConnectionPooling);
+
+    public boolean isUseSecuredHttp();
```

Comment:
Sounds like HTTPS, does this imply using the URL verification instead? I would suggest to call the property "isUseURLVerificationCheks" (not necessarily HTTP). Please change in all places where "secured HTTP" is used, it's confusing.